README.md

```diff
@@ -138,6 +139,18 @@ Options are set as attributes on the tooltip/popover components. Current tooltip
 - [spacing](#spacing)
 - [text (tooltip only)](#text)
 
+#### Animation duration
+
+| Type    | Number  |
+|---------|---------|
+| Default | 200     |
+
+Defines the duration of tooltip animation.
```

Comment:
We should specify that this is in `ms` and that in testing it will always be `0`.

Reply:
Done. Thanks for the quick response!